docs/src/content/docs/commands/JSON.STRLEN.md

```diff
@@ -1,43 +1,49 @@
 ---
 title: JSON.STRLEN
-description: Documentation for the DiceDB command JSON.STRLEN
+description: The 'JSON.STRLEN' command is used to get the length of a string at a given path in a JSON Document stored in DiceDB
 ---
 
 The `JSON.STRLEN` command is part of the DiceDBJSON module, which allows you to work with JSON data in DiceDB. This command is used to determine the length of a JSON string at a specified path within a JSON document stored in DiceDB.
 
 ## Syntax
 
-```plaintext
+```
 JSON.STRLEN <key> <path>
 ```
 
 ## Parameters
 
-- `key`: The key under which the JSON document is stored in DiceDB.
-- `path`: The JSONPath expression that specifies the location of the JSON string within the document. The path must point to a JSON string.
+| Parameter | Description                                                             | Type   | Required |
+|-----------|-------------------------------------------------------------------------|--------|----------|
+| `key`     | The key under which the JSON document is stored.                        | String | Yes      |
+| `path`    | The JSONPath to the string within the JSON document. The path must contain a string.  | String | No       |
 
 ## Return Value
 
-- `Integer`: The length of the JSON string at the specified path.
-- `Null`: If the path does not exist or does not point to a JSON string.
+| Condition                                 | Return Value                                                                    |
+|-------------------------------------------|---------------------------------------------------------------------------------|
+| JSON string is found at the specified path| Length of the JSON string.                                                      |
+| JSONPath contains `*` wildcard            | Indicates the length of each key in the JSON. Returns `nil` for non-string keys.|
+
 
 ## Behaviour
 
 When the `JSON.STRLEN` command is executed, DiceDB will:
 
-1. Retrieve the JSON document stored at the specified key.
-2. Navigate to the specified path within the JSON document.
-3. If the path points to a JSON string, return the length of the string.
-4. If the path does not exist or does not point to a JSON string, return `null`.
+1. If the specified `key` does not exist, the command returns `null`.
+2. If `path` is not specified, the command throws an error.
+3. `$` is considered as root path which returns `null`.
+4. If the specified path exists and points to a JSON string, the command returns the length of the string.
+5. If the path does not exist or does not point to a JSON string, an error is returned.
+6. Multiple results are represented as a list in case of wildcards, where each string length is returned in order of appearance and `null` is returned if it's not a string.
```

Comment:
Please verify if DiceDB returns null in these scenarios.

Reply:
![rootpath](https://github.com/user-attachments/assets/3701eef4-af89-455f-b3b2-bfc0dfe3b5e9)

Reply:
same as above